sphinxcontrib/confluencebuilder/config.py

```diff
@@ -123,4 +123,36 @@ def validate(c, log=True):
 properly set with the publisher's Confluence username.
 """)
 
+            if c.confluence_ca_cert:
+                if (not os.path.isfile(c.confluence_ca_cert) and
```

Comment:
you don't need parenthesis on this line

Reply:
Sorry, I'm used to the google style-guide. Would you prefer to see a backslash?